docs/developers/packaging.md

```diff
@@ -177,20 +181,20 @@ need to take into account to make this happen:
 * On macOS, the PKG installer does not offer a lot of flexibility for this configuration. We will
   put it under `~/Library/napari-<INSTALLER-VERSION>`, by default.
 
-This means that if you install `napari=0.4.16` using the installer, the actual `napari` executable
+This means that if you install {{ napari_conda_version }} using the installer, the actual `napari` executable
 can be found, by default, on the following locations:
 
-* Linux: `~/.local/napari-0.0.1/envs/napari-0.4.16/bin/napari`
-* MacOS: `~/Library/napari-0.0.1/envs/napari-0.4.16/bin/napari`
-* Windows: `~/napari-0.0.1/envs/napari-0.4.16/Library/bin/napari`
+* Linux: {{ '`~/.local/napari-0.0.1/envs/napari-NAPARI_VER/bin/napari`'.replace('NAPARI_VER', napari_version) }}
```

Comment:
It is really napari-0.0.1 not version number? As I have napari 0.4.17  bundle installed it store data in `napari-{version}/env/napari-{version}.